lua/lsp/providers/ruby.lua

```diff
@@ -0,0 +1,27 @@
+local opts = {
```

Comment:
Ruby is a special case.  

Currently, installing the language server using `:LspInstall` does not work correctly.  The language server, solargraph, doesn't like being launched from a different directory from the codebase and cannot find the project files.  

Solargraph needs to be gem installed `gem install solargraph` from the root directory of the project.  To get it started correctly, I have this in my configuration, which no longer works because the field `lsp` is no longer available on `lvim.lang.ruby.lsp`


```lua


local util = require("lspconfig/util")
lvim.lang.ruby.lsp.setup = {
	cmd = { "solargraph", "stdio" },
	filetypes = { "ruby" },
	init_options = {
		formatting = true,
	},
	root_dir = util.root_pattern("Gemfile", ".git"),
	settings = {
		solargraph = {
			diagnostics = true,
		},
	},
}
```

Reply:
So my plan is to port any generic changes upstream to lspconfig instead of maintaining them here, and only keep any edge-cases.
If you take a look here, you can see that you're basically want to use a generic setup: 
https://github.com/neovim/nvim-lspconfig/blob/aa0b9fd746d73a5ebbc72c732c645e96423d4504/lua/lspconfig/solargraph.lua

Can you try deleting the file entirely and see if solargraph works for you?